test/vmtester/tests.cpp

```diff
@@ -168,3 +168,67 @@ TEST_F(evmc_vm_test, set_tracer)
     if (vm->set_tracer)
         vm->set_tracer(vm, tracer_callback, nullptr);
 }
+
+TEST_F(evmc_vm_test, precompile_test)
+{
+    if (!evmc_vm_has_capability(vm, EVMC_CAPABILITY_PRECOMPILES))
+        return;
+
+    // Iterate every address (as per EIP-1352)
+    for (size_t i = 0; i < 0xffff; i++)
+    {
+        evmc_address destination{{0,
```

Comment:
Looks like giraffe. Maybe write it like:
```
auto destination = evmc_address{};
destination.bytes[18] = static_cast<uint8_t>(i >> 8);
...
```

Reply:
Its not a giraffe, but this guy:
![image](https://user-images.githubusercontent.com/20340/58190528-9e1a6c00-7cb4-11e9-8c18-aa295b68ae6d.png)